recipes/mo-catnip/meta.yml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mo-catnip" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/MetOffice/CATNIP/archive/1.0.tar.gz
+  sha256: 491e72214f2018dcd57e580956c97eb91b6df1cd6172b85a7e9e0da985733e54
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+
+  run:
+    - matplotlib-base >=2.2.3
+    - numpy >=1.15.4,<1.16.0
```

Comment:
Is there a reason to pin this below 1.16.0? The original packages doesn't seem to have this requirement.

Reply:
No, there is no such specific reason.